docs/en/api_reference/comment_spec_of_proto.md

```diff
@@ -18,30 +19,11 @@ message  BadCase{
 good case:
 
 ```
-message  GoodCase{
+// XXXXXXXX
+message GoodCase{
   // XXXXXXXX
   // XX
   // XXXXXX
   field A
 }
 ```
-
-Or you can use another annotation symbol directly `/*  */`
-
-
-If you want to have some comment in your proto files, but don't want them to be part of the docs, you can simply prefix the comment with `@exclude`.
-Example: include only the comment for the id field
-
-```
-/**
- * @exclude
- * This comment won't be rendered
- */
-message ExcludedMessage {
-  string id   = 1; // the id of this message.
-  string name = 2; // @exclude the name of this message
-
-  /* @exclude the value of this message. */
-  int32 value = 3;
-}
-```
```

Comment:
Why remove it? Can we keep this text?

Reply:
The reason is that `trailing comments` do not count towards passing `buf linter` comments rules.  
Maybe I can adding some tips to declare that `trailing comments` do not count towards passing `Proto Validation`.